qa/rpc-tests/mergetoaddress_helper.py

```diff
@@ -152,6 +153,11 @@ def run_test(self, test):
             "Destination address is also the only source address, and all its funds are already merged.",
             lambda: test.nodes[0].z_mergetoaddress([mytaddr], mytaddr))
 
+        # Merging will fail for this specific case where it would spend a fee and do nothing
```

Comment:
This is the wrong comment. (Sorry for not catching this earlier.)

Reply:
I missed this when working on this again too. It's poorly worded which makes it harder to catch.

Reply:
https://github.com/zcash/zcash/pull/4044